team-validator.js

```diff
@@ -177,7 +166,18 @@ class Validator {
 		if (format.onChangeSet) {
 			problems = problems.concat(format.onChangeSet.call(tools, set, format, setHas, teamHas) || []);
 		}
-		if (toId(set.species) !== template.speciesid) template = tools.getTemplate(set.species);
+
+		if (!template) {
+			template = tools.getTemplate(set.species);
+		}
+		if (!template || !template.abilities) {
+			set.species = 'Unown';
+			template = tools.getTemplate('Unown');
+		}
+		if (!template.exists) {
+			return [`The Pokemon "${set.species}" does not exist.`];
```

Comment:
Actually it's the Unown check that's redundant, and confused the validator - it started validating all unknown Pokémon as Unown, thus failing the test.